README.md

```diff
@@ -101,6 +101,7 @@ Clone this repository to your desired folder:
 
 - [ ] Add more methods
 - [ ] Add validations
+- [ ] Add views associated with the models and controllers already created.
```

Comment:
- Hey, you've not added test cases for your controllers. the test should be a test on request when moving to different routes in your app and the file should look something like this `spec/requests/posts_spec.rb`, and `spec/requests/users_spec.rb`. The test cases you have currently are not related to this milestone. please, kindly add some tests. check the project requirement to fully implement this.